HW2_Trofimov/calculator.py

```diff
@@ -0,0 +1,42 @@
+def main():
+    calc_input = input()
+    calc_input_lst = calc_input.split()
+
+    operation = calc_input_lst[1]
+    number_1 = calc_input_lst[0]
+    number_1 = float(number_1)
+    number_2 = calc_input_lst[2]
+    number_2 = float(number_2)
+
+    if operation == "+":
+        res = addition(number_1, number_2)
+    elif operation == "-":
+        res = subtraction(number_1, number_2)
+    elif operation == "*":
+        res = multiplication(number_1, number_2)
+    elif operation == "/":
+        res = division(number_1, number_2)
+    return print(res)
+
+
+def multiplication(number_1, number_2):
+    res = number_1 * number_2
+    return res
```

Comment:
Можно было бы это немного сократить: 
```suggestion
    return number_1 * number_2
```

Но это опять же про питонячесть, что понятие не шибко уловимое.